guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -42,38 +74,64 @@ Catalogs are compiled by a Puppet server from Puppet Manifests and data from Pup
 Candlepin:: A service within Katello responsible for subscription management.
 
 [[Compliance_policy]]
-Compliance policy:: Refers to a scheduled task executed on {ProjectServer} that checks the specified hosts for compliance against SCAP content.
+Compliance policy::
+Compliance policies refer to the application of SCAP content to hosts using {Project} with its OpenSCAP plugin.
+You can create compliance policies using the {ProjectWebUI}.
+Compliance reports require the setting of a specific XCCDF profile from a SCAP content, optionally using a tailoring file.
+You can setup scheduled tasks on {ProjectServer} that check your hosts for compliance against SCAP content.
```

Comment:
```suggestion
Compliance policies refer to the application of SCAP content to hosts by using {Project} with its OpenSCAP plugin.
You can create compliance policies by using the {ProjectWebUI}, Hammer CLI, or REST API.
A compliance policy requires the setting of a specific XCCDF profile from a SCAP content, optionally using a tailoring file.
You can set up scheduled tasks on {ProjectServer} that check your hosts for compliance against SCAP content.
When a compliance policy scan completes, the host sends an xref:ARF_report[ARF report] to {ProjectServer}.
```
I'm not sure if the second line about how you can create policies is necessary, but it certainly is not limited to web UI.

Reply:
Thanks! Applied and slightly reworded.